apps/functions-subscription/src/main.ts

```diff
@@ -70,3 +77,14 @@ app.http('getSubscription', {
   handler: makeGetSubscriptionHandler(env),
   route: '/trials/{trialId}/subscriptions/{userId}',
 });
+
+if (config.activations.job === 'on') {
+  app.cosmosDB('activationJob', {
+    containerName: 'activations',
+    connection: 'ActivationJobCosmosDBConnection',
+    databaseName: config.cosmosdb.databaseName,
+    handler: makeActivationCosmosDBHandler(),
+    createLeaseContainerIfNotExists: true,
```

Comment:
this will make a call to cosmosdb just to check if the leses container exists. if you are sure that the lease container exists (ie. you have created it with terraform which lets you set a specific througput), I suggest to remove this params to avoid a call.